packages/react-tabster/src/hooks/useNavigationAttribute.ts

```diff
@@ -0,0 +1,19 @@
+import { RefObject, useEffect, useRef } from 'react';
+import { KEYBOARD_NAV_ATTRIBUTE } from '../symbols';
+import { useTabster } from './useTabster';
+
+export const useNavigationAttribute = (): RefObject<HTMLElement> => {
```

Comment:
nit: Should this be called `useKeyboardNavigationAttribute`?

```suggestion
export const useKeyboardNavigationAttribute = (): RefObject<HTMLElement> => {
```